src/transcript-plugin.tsx

```diff
@@ -172,18 +173,20 @@ export class TranscriptPlugin extends KalturaPlayer.core.BasePlugin {
     return `${activeTextTrack?.language}-${activeTextTrack?.label}`;
   };
 
-  private _activatePlugin = () => {
+  private _activatePlugin = (isFirstOpen = false) => {
     this.ready.then(() => {
       this.sidePanelsManager?.activateItem(this._transcriptPanel);
       this._pluginState = PluginStates.OPENED;
       this.upperBarManager?.update(this._transcriptIcon);
+      this.dispatchEvent(TranscriptEvents.TRANSCRIPT_OPEN, {auto: isFirstOpen});
     });
   };
 
   private _deactivatePlugin = () => {
     this.ready.then(() => {
       this.sidePanelsManager?.deactivateItem(this._transcriptPanel);
       this.upperBarManager?.update(this._transcriptIcon);
```

Comment:
If we deactivated the plugin, should we not set the state to this._pluginState = PluginStates.CLOSED ?

Reply:
it seems that you are right, i guess @semarche-kaltura is most familiar with this plugin
anyway from experience... we rather not touch things unrelated directly to the original ticket